atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -234,11 +238,15 @@ private void runOnceForTable(TableToSweep tableToSweep) {
                     batchConfig,
                     startRow);
             long elapsedMillis = watch.elapsed(TimeUnit.MILLISECONDS);
-            log.debug("Swept {} unique cells from {} starting at {}"
+            log.info("Swept {} unique cells from {} starting at {}"
                             + " and performed {} deletions in {} ms"
                             + " up to timestamp {}.",
-                    results.getCellTsPairsExamined(), tableRef, startRowToHex(startRow),
-                    results.getStaleValuesDeleted(), elapsedMillis, results.getSweptTimestamp());
+                    SafeArg.of("", results.getCellTsPairsExamined()),
```

Comment:
`SafeArg.of("cellTsPairsExamined", results.getCellTsPairsExamined())`. In general please dont skip the parameter string, in the future internal logging product will use them for analysing logs.